pavement.py

```diff
@@ -206,8 +206,9 @@ def release(options):
 def compute_md5(idirs):
     released = paver.path.path(idirs).listdir()
     checksums = []
-    for f in sorted(released):
-        m = md5(open(f, 'rb').read())
+    for fn in sorted(released):
+        with open(fn, 'rb') as f:
+            m = md5(f.read())
```

Comment:
I don't know how to test these.
But the changes are small and easy to comprehend.

Reply:
You'd test this by installing `paver` and running `paver release`. But no need, this is clearly correct.